pupil-spa/src/app/services/audit/auditEntry.ts

```diff
@@ -1,6 +1,6 @@
-export type AuditEntryType = 'WarmupIntroRendered' | 'WarmupCompleteRendered' | 'CheckComplete' |
-'CheckStarted' | 'QuestionRendered' | 'QuestionAnswered' | 'PauseRendered' | 'RefreshDetected' |
-  'UtteranceStarted' | 'UtteranceEnded';
+export type AuditEntryType = 'WarmupIntroRendered' | 'WarmupCompleteRendered' | 'CheckStartedAPICallSucceed' |
+'CheckStartedAPICallFailed' | 'CheckComplete' |'CheckStarted' | 'QuestionRendered' | 'QuestionAnswered' |
```

Comment:
I am not sure about auditing CheckStartedApiCalledFailed and CheckStartedApiCallSucceed.  Why not just log CheckStarted?  We don't log any other call returns in the AuditLog, and it seems to be a lower level concern.

Reply:
@jon-shipley that was in the ticket's AC from Guy.